examples/express-sample/bin/www

```diff
@@ -65,11 +65,11 @@ function onError(error) {
   // handle specific listen errors with friendly messages
   switch (error.code) {
     case 'EACCES':
-      console.error(bind + ' requires elevated privileges');
+          console.error(`[Raygun4Node-Express-Sample] ` + bind + ` requires elevated privileges`);
```

Comment:
This extra spacing doesn't look correct. I feel `prettier` did not catch this. Could it be that code inside `bin/www/` is not checked by the code formatter?

Reply:
Yeah, it seems `prettier` triggers only for `.js` at the moment and not for the `www` script.

I'll PR that fix separately.

Reply:
Issue #204